src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -212,4 +212,5 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
       </div>
     </fieldset>
   </form>
+  <alm-work-item-link [workItem]="workItem" [loggedIn]="loggedIn"></alm-work-item-link>
```

Comment:
@mindreeper2420 for now it is acting as our "namespace" once we decide on a new name we can do a find/replace on it.